contracts/staking/PoSValidatorManager.sol

```diff
@@ -21,10 +21,10 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
     // solhint-disable private-vars-leading-underscore
     /// @custom:storage-location erc7201:avalanche-icm.storage.PoSValidatorManager
     struct PoSValidatorManagerStorage {
-        /// @notice The minimum amount of stake required to be a validator.
-        uint256 _minimumStakeAmount;
-        /// @notice The maximum amount of stake allowed to be a validator.
-        uint256 _maximumStakeAmount;
+        /// @notice The minimum weight of stake required to be a validator.
```

Comment:
I think the units for the PoS should be in staking amounts, since when users go to intiailize staking/delegation, it's with the stake amount units and we should keep consistent